SearchEngineCS/Phase5/Search.Test/QueryProcessorTest.cs

```diff
@@ -0,0 +1,29 @@
+using Moq;
+using Xunit;
+using SearchLibrary;
+using System.Collections.Generic;
+namespace Search.Test
+{
+    public class QueryProcessorTest
+    {
+        [Fact]
+        public void inputMock()
+        {
+            var mUserInput = new Mock<IUserInput>();
+            mUserInput.Setup(x => x.ScanInput()).Returns("hello +world -mamad");
+            var result = new HashSet<string>();
+
+            QueryProcessor query = new QueryProcessor(mUserInput.Object);
+            var ExpectedOrWord = new HashSet<string>();
+            ExpectedOrWord.Add("world");
```

Comment:
- Local variables must be **camelCase**
- You can write these two lines as below:
    ```csharp
    var expextedOrWord = new HashSet<string> { "world" };
    ```